CMakeLists.txt

```diff
@@ -15,19 +15,26 @@ if(POLICY CMP0074)
 endif()
 
 # Add cmake directory to CMake's path
-if(NOT CMAKE_MODULE_PATH)
-  set(CMAKE_MODULE_PATH)
-endif()
-list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
+list(APPEND CMAKE_MODULE_PATH
+    "${CMAKE_SOURCE_DIR}/cmake"
+    "${CMAKE_SOURCE_DIR}/cmake/external"
+)
+
+# Set some useful global variables
+# this will hold herbert/horace_on.m and worker_v2.m
+set(LOCAL_INIT_DIR "${CMAKE_BINARY_DIR}/local_init")
```

Comment:
should thie `LOCAL_INIT_DIR` be set in the HERBERT build environment too; it's set locally in `_test\CMakeLists.txt` as `${CMAKE_BINARY_DIR}/local_init"`

Reply:
Yes it should, I just haven't gotten around to it yet